parallax/parallax/core/python/mpi/graph_transform.py

```diff
@@ -38,9 +38,7 @@ def _add_aggregation_ops(gradients_info, op_to_control_consumer_ops, config):
         grad = grad_tensor
         grad_consumers = [c for c in grad.consumers()]
         agg_grad = hvd.allreduce(grad,
-                                 average_dense=True,
-                                 average_sparse=config.average_sparse,
```

Comment:
Setting `config.average_sparse` parameter should also be omitted in each `parallax_config.py` file of example codes.

Reply:
The argument is still used at line 53 for sparse gradients(tf.IndexedSlices).